### DISCUSSION TITLE
設置 Sitemap 優先等級

### DISCUSSION THREAD

**Discussion by wastu01** [CaiJimmy/hugo-theme-stack#906] [Q&A] [answered]
可以在每個文章 frontmatter 中可以設定嗎？

例如：priority = 0.7
 `<priority>0.7</priority>`

還是必須得手動更改 sitemap ?

**Comment by CaiJimmy** [comment, marked as answer]
主题没有自定义 Hugo 的 Sitemap 模板，请参考官方文档：https://gohugo.io/templates/sitemap-template/#override-default-values
